CMakeLists.txt

```diff
@@ -99,6 +99,27 @@ if(JACK)
   install(FILES cmake/modules/FindJACK.cmake DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/portaudio/modules")
 endif()
 
+# PulseAudio is mostly for Linux
+
+find_package(PULSEAUDIO)
+cmake_dependent_option(PULSEAUDIO "Enable support for PulseAudio general purpose sound server" ON PULSEAUDIO_FOUND OFF)
+if(PULSEAUDIO)
+  target_link_libraries(PortAudio PRIVATE ${PULSEAUDIO_LIBRARIES})
+  target_sources(PortAudio PRIVATE src/hostapi/pulseaudio/pa_linux_pulseaudio_block.c
+    src/hostapi/pulseaudio/pa_linux_pulseaudio.c
+    src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c)
+
+  set(PORTAUDIO_PUBLIC_HEADERS "${PORTAUDIO_PUBLIC_HEADERS}" src/hostapi/pulseaudio/pa_linux_pulseaudio_block_internal.h
+    src/hostapi/pulseaudio/pa_linux_pulseaudio_internal.h
+    src/hostapi/pulseaudio/pa_linux_pulseaudio_cb_internal.h)
```

Comment:
```suggestion
  set(PORTAUDIO_PUBLIC_HEADERS "${PORTAUDIO_PUBLIC_HEADERS}"
    src/hostapi/pulseaudio/pa_linux_pulseaudio_block_internal.h
    src/hostapi/pulseaudio/pa_linux_pulseaudio_internal.h
    src/hostapi/pulseaudio/pa_linux_pulseaudio_cb_internal.h)
```

Reply:
Wait, these don't look like public headers. Headers in PORTAUDIO_PUBLIC_HEADERS get installed to CMAKE_INSTALL_INCLUDEDIR and should be in the `include` directory at the top of the repository.

Reply:
include/pa_linux_pulseaudio.h is the only file that should be added to this list for PulseAudio

Reply:
Sorry I didn't read these before pushing commit.. I'll handle this later on.